crap-advisor-frontend/src/core/components/Sidebar/index.css

```diff
@@ -1,27 +1,76 @@
 .sidebar {
-    padding: 20px;
     font-size: 14px;
-    background-color: white;
-    width: 300px;
-    border-right: 1px solid black;
     z-index: 1000;
+    box-shadow: 0 0 0 2px rgb(0, 0, 0, 0.1);
+    overflow-y: auto;
 }
 
-.sidebar-restroom-header {
-    font-size: 32px;
+.sidebar::-webkit-scrollbar {
+    width: 5px;
 }
 
-.sidebar-restroom-rating {
-    font-size: 16px;
+.sidebar::-webkit-scrollbar-thumb {
+    background: rgba(0, 0, 0, 0.1);
+}
+
+.sidebar::-webkit-scrollbar-thumb:hover {
+    background: rgba(0, 0, 0, 0.15);
 }
 
-.sidebar-review-container {
-    border: 1px solid black;
-    border-radius: 4px;
-    padding: 5px;
-    margin-top: 5px;
+.sidebar::-webkit-scrollbar-thumb:active {
+    background: rgba(0, 0, 0, 0.2);
 }
 
 .logo {
-    width: 300px;
+    font-weight: 300;
+    text-align: center;
+    color: #cbcbcb;
+}
+
+.welcome {
+    font-size: 16px;
+    font-weight: 400;
+    text-align: center;
+}
+
+.section-restroom {
+    --hue: 200;
```

Comment:
As good practice, you should create a main css file, where you can add global variables (for exaple, in `#root` element)

Reply:
My previous comment is apply to every HEX in your css code too. You can stay variables inside main.css file or create globals.css file

Reply:
I've moved some styles to main.css as you said